client/my-sites/stats/activity-log-day/index.jsx

```diff
@@ -115,6 +115,7 @@ class ActivityLogDay extends Component {
 		} = this.props;
 
 		const formattedDate = applySiteOffset( moment.utc( tsEndOfSiteDay ) ).format( 'LL' );
+		const noActivityText = isToday ? translate( 'No activity yet' ) : translate( 'No activity' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **84** times:
`translate( 'No activity yet!' )` *ES Score: 14*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/activity-log/future-days)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
@keoshi how do you feel about reusing `'No activity yet!'` (with the exclamation)?

Reply:
@sirreal missed this comment, sorry. I don't mind it, and if we're getting more consistency in the process better yet!

Reply:
f247a13 👍